clients/src/test/java/org/apache/kafka/test/TestSslUtils.java

```diff
@@ -56,6 +56,24 @@
 
 public class TestSslUtils {
 
+    /*
+     * Simulates a CA signing certificates
+     */
+    static KeyPair issuerKP;
+    static AsymmetricKeyParameter privateKeyAsymKeyParam;
+    static X509Certificate issuerCert;
+    static X500Name issuer = new X500Name("CN=localhost, O=CAroot");
```

Comment:
Maybe the 3 variables above should be named `caPrivateKeyParam`, `caCert` and `caName`? Also, maybe they should be private.